store/persistent/benchmarks.go

```diff
@@ -105,6 +107,76 @@ func BenchParallelGet(s store.StorageFlusher, size string, b *testing.B) {
 	wg.Wait()
 }
 
+func BenchCidGet(s store.PersistentStorage, b *testing.B) {
+	cids, err := utils.RandomCids(1)
+	if err != nil {
+		panic(err)
+	}
+	p, _ := peer.Decode("12D3KooWKRyzVWW6ChFjQjK4miCty85Niy48tpPV95XdKu1BcvMA")
+
+	entry := store.MakeIndexEntry(p, protocolID, cids[0].Bytes())
+
+	cids, _ = utils.RandomCids(4096)
+	s.PutMany(cids, entry)
+
+	for testCount := 1024; testCount < 10240; testCount *= 2 {
```

Comment:
Each test run is changing the number of queries done, but so is `b.N`.  I see that the intent is to benchmark how long it takes to do some number of queries, and each test run changes the number of queries.  However, effectively what is being done is to make the benchmark less sensitive / less accurate with each run, since the same values of `b.N` are not used by different runs.  The benchmarker changes `b.N` only as much as it needs to in order to get to a stable benchmark.

If you use `b.N` as the limit in the outer loop, then you can get a benchmark of how long it takes to do some fixed number of queries.